jsdoc.json

```diff
@@ -1,96 +1,100 @@
-        "node_modules/jsdoc-escape-at",
-        "better-docs/typedef-import"
+  "tags": {
+    "allowUnknownTags": true
+  },
+  "source": {
+    "include": [
+      "./adminjs/src",
+      "./adminjs-hapijs",
+      "./adminjs-mongoose/src",
+      "./adminjs-sequelizejs",
+      "./adminjs-typeorm/src",
+      "./adminjs-expressjs",
+      "./adminjs-firebase-functions",
+      "./adminjs-upload/src",
+      "./adminjs-passwords/src",
+      "./adminjs-design-system/src",
+      "./adminjs-nestjs/src",
+      "./adminjs-prisma"
     ],
-    "opts": {
-        "encoding": "utf8",
-        "destination": "docs",
-        "readme": "./docs-src/home.md",
-        "recurse": true,
-        "private": true,
-        "verbose": true,
-        "tutorials": "./docs-src/tutorials",
-        "sections": "./docs-src/sections",
-        "template": "better-docs"
+    "includePattern": "\\.(jsx|js|ts|tsx)$",
+    "excludePattern": "(node_modules/|docs|(bundle.*\\.js)|cli.ts)"
+  },
+  "plugins": [
+    "plugins/markdown",
+    "jsdoc-mermaid",
+    "better-docs/navigation",
+    "better-docs/component",
+    "node_modules/jsdoc-typeof-plugin",
+    "better-docs/typescript",
+    "node_modules/jsdoc-escape-at",
+    "better-docs/typedef-import"
+  ],
+  "opts": {
+    "encoding": "utf8",
+    "destination": "docs",
+    "readme": "./docs-src/home.md",
+    "recurse": true,
+    "private": true,
+    "verbose": true,
+    "tutorials": "./docs-src/tutorials",
+    "sections": "./docs-src/sections",
+    "template": "better-docs"
+  },
+  "templates": {
+    "cleverLinks": false,
+    "monospaceLinks": false,
+    "search": true,
+    "default": {
+      "includeDate": false,
+      "staticFiles": {
+        "include": [
+          "./docs-src/statics",
+          "./adminjs/src/frontend/assets/styles",
+          "./adminjs/src/frontend/assets/fonts"
+        ]
+      }
     },
-    "templates": {
-        "cleverLinks": false,
-        "monospaceLinks": false,
-        "search": true,
-        "default": {
-            "includeDate": false,
-            "staticFiles": {
-              "include": [
-                  "./docs-src/statics", "./adminjs/src/frontend/assets/styles", "./adminjs/src/frontend/assets/fonts"
-              ]
-            }
+    "better-docs": {
+      "landing": "./docs-src/landing.html",
+      "logo": "images/logo-reg.svg",
+      "title": "AdminJS - Auto-generated admin panel for node.js with react",
+      "component": {
+        "wrapper": "./adminjs/docs/component.tsx"
+      },
+      "trackingCode": "<!-- Google Tag Manager --><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-5L24P7Z');</script><!-- End Google Tag Manager -->",
```

Comment:
Can you add a new GTM:
```
<!-- Google Tag Manager -->
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-K6R3HZ4');</script>
<!-- End Google Tag Manager -->
```

There's also a second part of GTM which should go into `body`:
```
<!-- Google Tag Manager (noscript) -->
<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-K6R3HZ4"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<!-- End Google Tag Manager (noscript) -->
```

Reply:
I added a new property bodyTrackingCode and put it within body tags